terraform/modules/cds_export/bucket.tf

```diff
@@ -8,7 +8,7 @@ resource "aws_s3_bucket" "this" {
   #checkov:skip=CKV_AWS_145: default encryption is fine
   #checkov:skip=CKV_AWS_144: cross region replication is overkill when this is only for data transfer
   #checkov:skip=CKV_AWS_21: versioning not important, data source is elsewhere
-  bucket = "${var.prefix}-export"
+  bucket = "${var.prefix}-export${var.bucket_suffix}"
```

Comment:
Annoyingly this caused a duplicate name error without an extra suffix - I didn't want to change the prefix for this module, so the optional suffix seemed to be a reasonable answer.